packages/checkbox/src/react/__specs__/index.spec.tsx

```diff
@@ -23,4 +23,15 @@ describe('Checkbox', () => {
 
     expect(ref.current).not.toBeNull()
   })
+
+  it('is accessible by role', () => {
+    const { getByRole } = render(
+      <>
+        <Checkbox label="First" name="first" value="value" />
+        <Checkbox label="Second" name="second" value="value" />
+      </>
+    )
+
+    expect(getByRole('checkbox', { name: /first/i })).toBeInTheDocument()
```

Comment:
@willsoto i needed to use regex to query the accessibility tree. couldn't do it just by string ie `{ name: 'first' }`